cves/CVE-2003-0083.yml

```diff
@@ -174,8 +175,9 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: If they didn't trust the input for access logs, they would have been
+          more careful santitizing input.
```

Comment:
I think this sounds a little harsh, you could word this differently

Reply:
Changed